source/users.yaml

```diff
@@ -154,6 +154,41 @@
             additionalProperties:
               $ref: "#/definitions/User"
 
+  '/users/search':
+    post:
+      tags:
+        - users
+      summary: Search users
+      description: Get a map of sanitized user objects by search parameter
+      parameters:
+        - in: body
+          name: body
+          description: Search parameter
+          required: true
+          schema:
+            type: object
+            required:
+              - term
+              - team_id
```

Comment:
`team_id` is actually optional, when it's blank or missing the search is performed across all teams